### ISSUE TITLE
Add ability to include user's border/shadow to image transforms

### ISSUE THREAD

**Issue by doobox** [yourhead/s3#19]
When creating %image% clones, via the plist image transform method, user image adjustments are not carried through to the new cloned images.

As a stacks developer may not want the image adjustments in the new cloned images, there should be a plist key to optionally add user adjustments to the cloned images.

Also as a side note, meta data is not carried through to the clones either.
I'd guess there would be no implications to bringing the meta data into image clones, so would not require a plist optional key.

**Comment by yourhead** [comment]
copied from slack chat:
 i see what you mean **in theory** but in practice I'm still not sure it makes sense.

the user adjustments offer a few things:  format, size, borders, and shadows.  overriding the size and format makes sense to fit a layout.

but that leaves only the border and shadow. those don't scale easily, especially if you were to change the aspect ratio.

to create @2x images in S3, i double all the borders and shadows. but this only works because i know the image will be presented in exactly the same space, but twice the dpi.

in other words:  i feel like this is a feature that **seems** like a good idea -- until you actually try it in practice -- and then it would seem silly.

and...   the added borders and shadows of stacks is seeming like a feature who's time has passed.  css is now a much more efficient and flexible way to give those things to users in 99% of the cases -- i think we should be focusing our efforts on that.
